plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -69,6 +89,14 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
 
     // TODO move common tests from connector-specific classes here
 
+    private final ExecutorService threadPool = newCachedThreadPool(daemonThreadsNamed("base-connector-test"));
```

Comment:
threadPool -> executor

add "jdbc-" to thread names ... or maybe derived the name from `getClass().getName()`